framework/controller/controller_test.go

```diff
@@ -1285,7 +1285,6 @@ func TestViewUnnamed(t *testing.T) {
 	is.NoErr(err)
 	is.NoErr(res.DiffHeaders(`
 		HTTP/1.1 200 OK
-		Transfer-Encoding: chunked
```

Comment:
I'm quite surprised the changes affected these tests. Do you know why you needed to remove this?

Reply:
Reverted. But I still get an error on it while running the unit test. I use `go 1.7` and `node 18.9`